tests/main/refresh-devmode/task.yaml

```diff
@@ -36,6 +35,9 @@ restore: |
     teardown_store $STORE_TYPE $BLOB_DIR
 
 execute: |
+    # skip the test for nonproduction stores on ubuntu-core systems
+    if [ "$STORE_TYPE" = "fake" -a "$SPREAD_SYSTEM" =~ ubuntu-core-16-* ]; then exit; fi
```

Comment:
Would read better in three lines rather than one. Can even drop the comment, since it's just re-stating exactly what the code is doing.